src/components/DayPickerRangeController.jsx

```diff
@@ -247,28 +249,39 @@ export default class DayPickerRangeController extends React.Component {
       focusedInput,
       renderDay,
       renderCalendarInfo,
+      calculateToday,
     } = this.props;
 
     const modifiers = {
-      today: day => this.isToday(day),
       blocked: day => this.isBlocked(day),
       'blocked-calendar': day => isDayBlocked(day),
       'blocked-out-of-range': day => isOutsideRange(day),
       'blocked-minimum-nights': day => this.doesNotMeetMinimumNights(day),
       'highlighted-calendar': day => isDayHighlighted(day),
       valid: day => !this.isBlocked(day),
-      // before anything has been set or after both are set
-      hovered: day => this.isHovered(day),
-
-      // while start date has been set, but end date has not been
-      'hovered-span': day => this.isInHoveredSpan(day),
-      'after-hovered-start': day => this.isDayAfterHoveredStartDate(day),
-      'last-in-range': day => this.isLastInRange(day),
 
       // once a start date and end date have been set
       'selected-start': day => this.isStartDate(day),
```

Comment:
What if we did something similar for the selection related ones?

`selected-start` requires `START_DATE` to exist,
`selected-end`, and `blocked-minimum-nights require `END_DATE`,
`selected-span`, `last-in-range` require both,

we could apply those conditionals within the touch params as well.